### DISCUSSION TITLE
OK what's a "Broken pipe"  ???

### DISCUSSION THREAD

**Discussion by Teledyol** [Chia-Network/chia-blockchain#2281] [Q&A] [answered]
…hope it's not sewage

06:45:55.732 full_node full_node_server        : WARNING  [Errno 32] Broken pipe 222.64.106.216
06:46:28.005 full_node full_node_server        : WARNING  [Errno 32] Broken pipe 115.179.192.127
06:57:17.197 full_node full_node_server        : ERROR    Exception Was not able to get peername from <function WSChiaConnection.__getattr__.<locals>.invoke at 0x104a38940> at 180.140.125.80, exception Stack: Traceback (most recent call last):
  File "chia/server/server.py", line 229, in incoming_connection
  File "chia/server/ws_connection.py", line 59, in __init__
ValueError: Was not able to get peername from <function WSChiaConnection.__getattr__.<locals>.invoke at 0x104a38940> at 180.140.125.80

07:06:02.669 full_node full_node_server        : ERROR    Exception Was not able to get peername from <function WSChiaConnection.__getattr__.<locals>.invoke at 0x10f68b3a0> at 60.25.250.186, exception Stack: Traceback (most recent call last):
  File "chia/server/server.py", line 229, in incoming_connection
  File "chia/server/ws_connection.py", line 59, in __init__
ValueError: Was not able to get peername from <function WSChiaConnection.__getattr__.<locals>.invoke at 0x10f68b3a0> at 60.25.250.186

07:17:07.601 full_node full_node_server        : WARNING  [Errno 32] Broken pipe 163.125.213.51
07:35:40.992 full_node full_node_server        : WARNING  [Errno 32] Broken pipe 39.183.155.66
07:37:04.488 full_node full_node_server        : ERROR    Exception Was not able to get peername from <function WSChiaConnection.__getattr__.<locals>.invoke at 0x10cb51f70> at 123.174.74.214, exception Stack: Traceback (most recent call last):
  File "chia/server/server.py", line 229, in incoming_connection
  File "chia/server/ws_connection.py", line 59, in __init__
ValueError: Was not able to get peername from <function WSChiaConnection.__getattr__.<locals>.invoke at 0x10cb51f70> at 123.174.74.214

**Comment by hoffmang9** [comment, marked as answer]
That is normal and just a peer disconnecting from you.

**Reply by Teledyol** [reply]
Thanks

**Comment by tartley** [comment]
If it's normal, should it log an ERROR with traceback?
